File: app/src/main/java/com/sleticalboy/dailywork/util/ObjectUtils.java
package com.sleticalboy.dailywork.util;

/**
 * Created on 18-2-7.
 *
 * @author sleticalboy
 * @version 1.0
 * @description
 */
public class ObjectUtils {

    public static boolean equals(Object actual, Object excepted) {
        return actual == excepted || (actual == null ? excepted == null : actual.equals(excepted));
    }
}

File: MainApp/src/main/java/com/sleticalboy/dailywork/util/ObjectUtils.kt
package com.sleticalboy.dailywork.util

/**
 * Created on 18-2-7.
 *
 * @author sleticalboy
 * @version 1.0
 * @description
 */
object ObjectUtils {

    fun equals(actual: Any?, excepted: Any?): Boolean {
        return actual === excepted || if (actual == null) excepted == null else actual == excepted
    }
}
